xml/depl_crowbar.xml

```diff
@@ -71,29 +71,24 @@
   <procedure xml:id="pro.depl.crow.password">
    <title>Changing the Password for the &crow; Web Interface</title>
    <step>
-    <para>After logging in to the &crow; &wi;, select <menuchoice>
-      <guimenu>&Barcl;s</guimenu>
-      <guimenu>Crowbar</guimenu>
-     </menuchoice>.</para>
-   </step>
-   <step><para>Select the <literal>&crow;</literal> &barcl; entry and
-    <guimenu>Edit</guimenu> the proposal.</para>
-   </step>
-   <step>
-    <para>In the <guimenu>Attributes</guimenu> section, click
-      <guimenu>Raw</guimenu> to edit the configuration file.</para>
+    <para>
+     On the &admserv;, open the following file in a text editor:
+     <filename>/etc/crowbarrc</filename>. Its content looks like:
```

Comment:
It contains the following: (Instead of "Its content looks like" - just reads a bit clunky)